src/main/java/frc/robot/Robot.java

```diff
@@ -1,17 +1,45 @@
 package frc.robot;
 
 import com.chopshop166.chopshoplib.commands.CommandRobot;
+import com.chopshop166.chopshoplib.controls.ButtonXboxController;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.maps.RobotMap;
+import frc.robot.subsystems.Climber;
 
 public class Robot extends CommandRobot {
 
+  private Command autonomousCommand;
+
+  private final SendableChooser<Command> autoChooser = new SendableChooser<>();
+
+  private final ButtonXboxController controller = new ButtonXboxController(0);
+
+  private final RobotMap map = getMapForName("Gaston", RobotMap.class, "frc.robot.maps");
+
+  private final Climber leftClimber = new Climber(map.getLeftTelescopeMap());
+  private final Climber rightClimber = new Climber(map.getRightTelescopeMap());
+
   @Override
   public void robotInit() {
-    // Nothing to initialize
+
```

Comment:
This is definitely a mistake on my part, but please have `robotInit` call `super.robotInit()` inside it. Otherwise nothing will run.